run/hello-broken/e2e_test.py

```diff
@@ -23,86 +23,132 @@
 
 import pytest
 
+# Unique suffix to create distinct service names
+SUFFIX = uuid.uuid4().hex
```

Comment:
Since we're rewriting this anyway, we might as well only grab the first ~6 digits of the uuid.  If cloud run service names get too long, they'll fail.

Reply:
I'm suspecting that's why I'm getting these test failures and just hadn't bothered to debug yet. Thank you!